docs/writing-your-first-rclrs-node.md

```diff
@@ -0,0 +1,224 @@
+# Writing your first `rclrs` node
+
+This tutorial is intended to point out some of the differences of the Rust client library with the other client libraries. It assumes knowledge of Rust, and is also not intended to be an introduction to ROS 2.
+
+As a semi-realistic example, let's create a node that periodically republishes the last message it received. It's limited to only one specific message type – `std_msgs/msg/String` in this example.
+
+## Create a package
+
```

Comment:
Would be good to use the standard ROS2 tools for creating packages:

```
ros2 pkg create --build-type ament_cargo <package_name>
```

More info at https://docs.ros.org/en/galactic/Tutorials/Creating-Your-First-ROS2-Package.html#create-a-package

Though this may require some sort of integration with `ros2` so that it's gets expanded with actual Rust code. Haven't looked up how to wire up `ros2` with build types.

Reply:
@esteve I agrere that would be good, but let's focus on the tutorial article here please. Doing that integration isn't the goal of this PR.

Reply:
@nnmm I didn't suggest working on this integration in this PR, but thought it was appropriate to point out that the ROS2 way of creating a new package is via `ros2 pkg create`, not via creating files manually. Please add a comment to the tutorial referencing `ros2 pkg create`, but pointing out that until the integration is done, users will have to edit files manually.

Reply:
@nnmm thanks :slightly_smiling_face: